docs/security/multisig.md

```diff
@@ -0,0 +1,49 @@
+# Multisig
+
+## How it works
+
+Yearn's governance is implemented by a 6-of-9 multi-signature wallet. The members of the multi-signature wallet were voted in by YFI holders and are subject to change from future governance votes. More information about Yearn governance and how it interacts with the multisig can be found on [YIP-61: Governance 2.0](https://gov.yearn.finance/t/yip-61-governance-2-0/10460) and on the [FAQ](../resources/faq#governance).
```

Comment:
I'm being picky here. Yearn's governance is ruled by Gov 2.0. The multisig is just a part of it and has some powers delegated to it by YFI holders